R/impact_central.R

```diff
@@ -4,6 +4,8 @@ get_raw_impact_details <- function(con, meta1, burden_outcome, is_under5 = FALSE
   stopifnot(burden_outcome %in% c("deaths", "cases", "dalys", "yll",
                                   "deaths_cwyx", "cases_cwyx", "dalys_cwyx", "yll_cwyx")) # _cwyx outcomes are MenA specific.
   stopifnot(is_under5 %in% c(TRUE, FALSE))
+  meta1$burden_outcome <- gsub("rubella_deaths_congenital", "deaths", meta1$burden_outcome)
```

Comment:
on the note of rubella, is the impact reassigned to 0 year olds when age structuring the calendar year impact within vimpact?

Reply:
Nope. This is not doing the job. vimpact only takes estimates as is. Reassign impact to 0yos shall be post-hoc.